Project.toml

```
@@ -3,7 +3,7 @@ uuid = "7a57a42e-76ec-4ea3-a279-07e840d6d9cf"
 keywords = ["probablistic programming"]
 license = "MIT"
 desc = "Common interfaces for probabilistic programming"
-version = "0.6.3"
+version = "0.7.1"
```

If this is the first breaking release, then this should be
```suggestion
version = "0.7.0"
```